index.js

```diff
@@ -235,6 +238,28 @@ function logError(context, error){
     data.previousPeriod = json.settings[0].value;
   }
 
+  if (argv.qualityGateStatus === 'true') {
+      try {
+          const response = await got(`${sonarBaseURL}/api/qualitygates/project_status?projectKey=${sonarComponent}`, {
```

Comment:
Can you also add the branch and pull request Id inside this query parameters if they are set in flag pls ?